cmd/xl-v1-multipart-common.go

```diff
@@ -19,14 +19,13 @@ package cmd
 import (
 	"path"
 	"sync"
+	"time"
 )
 
-// writeUploadJSON - create `uploads.json` or update it with change
-// described in uCh.
-func (xl xlObjects) updateUploadJSON(bucket, object string, uCh uploadIDChange) error {
+// updateUploadJSON - add or remove upload ID info in all `uploads.json`.
+func (xl xlObjects) updateUploadJSON(bucket, object, uploadID string, initiated *time.Time, isRemove bool) error {
```

Comment:
If its not `*time.Time`, then removeUploadID() would need to send `time.Time{}` which is not necessary.  Currently it calls `updateUploadJSON()` with `nil`